app/views/checkout/thankyou.scala.html

```diff
@@ -58,18 +58,21 @@ <h2 class="section-title">What happens now?</h2>
 
             <div class="steps">
                 <h4 class="steps__title">We’ll send you an email confirming your subscription and payment information.</h4>
-                @if(plan.isHomeDelivery) {
-                    <h4 class="steps__title">Your first paper will be delivered on @startDate. </h4>
-                }
                 @if(plan.isVoucher) {
                     <h4 class="steps__title">You will then receive your personalised book of vouchers.</h4>
                     <h4 class="steps__title">Take your vouchers to your newsagent or where you buy your paper.</h4>
                 }
+                @if(guestAccountForm.isDefined) {
+                    <h4 class="steps__title">You can enter a password to manage your subscription online:</h4>
```

Comment:
@JuliaBellis and I think this to be suggestive rather than didactic, given it's not a hard requirement on the user. Taking the vouchers to a news agent is, and so it makes that stand out.